simulator/src/resim/addressing.rs

```diff
@@ -147,11 +149,44 @@ impl FromStr for SimulatorNonFungibleAddress {
     type Err = ParseNonFungibleAddressError;
 
     fn from_str(address: &str) -> Result<Self, Self::Err> {
```

Comment:
Possibly some unit test to show case the syntax?

Reply:
Will add that!

Reply:
I have added that. Here is what the syntax for `NonFungibleAddress`es in resim looks like now: https://github.com/radixdlt/radixdlt-scrypto/blob/8ebe250c4ba3ec33b6bcc3a2aab014949bffe50a/simulator/src/utils/prepare_instruction.rs#L853